docker/dev/local-http-signatures/app.js

```diff
@@ -0,0 +1,56 @@
+const Koa = require('koa')
```

Comment:
just turned it into a js app to remove all of the TS dependencies and build step since it's so small

Reply:
@sabineschaller I don't know if there is a cleaner way to have this here without publishing a docker image